rosdep/base.yaml

```diff
@@ -6290,6 +6290,11 @@ wireless-tools:
   fedora: [wireless-tools]
   gentoo: [net-wireless/wireless-tools]
   ubuntu: [wireless-tools]
+wireshark:
+  debian: [wireless-tools]
+  fedora: [wireless-tools]
```

Comment:
These both look like a typo; I think they should probably be `wireshark` instead.